features/include/pcl/features/impl/integral_image_normal.hpp

```diff
@@ -1047,7 +1047,7 @@ pcl::IntegralImageNormalEstimation<PointInT, PointOutT>::computeFeaturePart (con
           continue;
         }
 
-        if (u < border || v > right)
+        if (u < border || u > right)
```

Comment:
I wish there was a simpler way to write this check using `u` just once. No issues, just commenting

```cpp
abs(2*u - width) >= 2*(width - 2*border)
// or a function cal
```